packages/core/admin/ee/server/services/review-workflows/review-workflows.js

```diff
@@ -35,18 +35,26 @@ async function initDefaultWorkflow({ workflowsService, stagesService, strapi })
 
 function extendReviewWorkflowContentTypes({ strapi }) {
   const extendContentType = (contentTypeUID) => {
-    const setStageAttribute = set(`attributes.${ENTITY_STAGE_ATTRIBUTE}`, {
-      writable: true,
-      private: false,
-      configurable: false,
-      visible: false,
-      useJoinTable: true, // We want a join table to persist data when downgrading to CE
-      type: 'relation',
-      relation: 'oneToOne',
-      target: 'admin::workflow-stage',
-    });
-    strapi.container.get('content-types').extend(contentTypeUID, setStageAttribute);
+    const setAttribute = (path, target) =>
```

Comment:
Nice, thanks for handling this as well in this PR!

I think this comment is relevant here too - https://github.com/strapi/strapi/pull/16889#discussion_r1218198223